File: v2/golang/auth_test.go
package golang

import (
	"fmt"
	"testing"
)

func TestGetAuthToken(t *testing.T) {
	token, err := GetAuthToken("http://localhost:9080", "admin", "admin")
	if err != nil {
		t.Error(err)
		return
	}
	fmt.Printf("Token: %v.\n", token)
}

File: v2/java-apache-httpclient/src/test/java/com/striim/restsample/AuthTest.java
package com.striim.restsample;

import org.junit.Test;
import static org.junit.Assert.*;

/**
 *
 * @author subhash
 */
public class AuthTest {

    /**
     * Test of getAuthToken method, of class Auth.
     */
    @Test
    public void testGetAuthToken() throws Exception {
        System.out.println("getAuthToken");
        String url = "http://localhost:9080";
        String uname = "admin";
        String pwd = "admin";
        int expResultSize = 36;
        String result = Auth.getAuthToken(url, uname, pwd);
        System.out.println("Token: " + result);
        assertEquals(expResultSize, result.length());
    }
    
}
